src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -898,20 +903,20 @@ static void _handlePaintAttr(SvgPaint* paint, const char* value)
         paint->none = true;
         return;
     }
-    paint->none = false;
```

Comment:
Just double-checking., is this correct? It seems to be required for the condition `currentColor`.

Reply:
I modified it.